src/repl/QuackRepl.php

```diff
@@ -101,24 +127,50 @@ function readline_callback($command)
             goto next;
     }
 
-    $lexer = new Tokenizer($command);
+    $run_command = !$session->complete_stmt
```

Comment:
Prefer not using negations on conditions. The pattern `cond ? pass : fail` is easier to read than `!cond ? fail : pass`. :smile: